teamscale-maven-plugin/pom.xml

```diff
@@ -0,0 +1,170 @@
+<project>
+    <modelVersion>4.0.0</modelVersion>
+
+    <properties>
```

Comment:
I am a huge fan of running `mvn org.codehaus.mojo:tidy-maven-plugin:1.1.0:pom` to auto-format POMs according to the  . If you are paranoid, you can even make https://www.mojohaus.org/tidy-maven-plugin/check-mojo.html  part of your build.